packages/app/src/app/store/modules/deployment/actions.js

```diff
@@ -14,6 +15,51 @@ export function loadZip({ props, jsZip }) {
   return jsZip.loadAsync(file).then(result => ({ contents: result }));
 }
 
+export function getPeopleWhoWant2() {
```

Comment:
Could we perhaps make this function async and dynamically import `airtable`? That should shave us some bundle size 😄

Reply:
You want a fancy require inside that ? 😄